.cypress/integration/trace_analytics_test/trace_analytics_traces.spec.js

```diff
@@ -125,32 +125,34 @@ describe('Testing traces table', () => {
     setTimeFilter();
   });
 
-  it('Renders the traces table and verify Table Column, Pagination and Rows Data ', () => {
+  it.only('Renders the traces table and verify Table Column, Pagination and Rows Data ', () => {
```

Comment:
I believe `.only` will only run this test in the suite

Reply:
Good catch thank you forgot to remove